docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
-  for more information. Not required if the private key is part of the file
-  defined in ``--certfile``. Not allowed if ``--certfile`` option not provided.
-  Default: No client key file. Client private key should then be part of the
-  file defined by ``--certfile``.
-* **--output-format/-o** - Output format choice (Default: mof).
-  Note that the actual output format may differ from this value because some
-  commands only allow selected formats.  See :ref:`Output formats` for
-  detailed information on the output formats.
-* **--use-pull-ops** [ ``yes`` | ``no`` | ``either`` ] - Determines
-  whether the pull operations are used for ``EnumerateInstances``,
-  ``AssociatorInstances``, ``ReferenceInstances``, and ``ExecQuery`` operations,
-  See :ref:`Pywbemcli and the DMTF pull operations` for more information on
-  pull operations:
-
-  * ``yes`` -  pull requests will be used and if the server does not
-    support pull, the operation will fail.
-  * ``no`` - forces pywbemcli to try only the traditional non-pull operations.
-  * ``either`` - (default) pywbem tries both; first pull and then traditional
-    operations.
-
-* **--pull-max-cnt** -  controls the CIM-XML open and pull request parameter
-  ``MaxObjectCount`` that tells the server the maximum number of objects
-  to be returned for each pull request if pull operations are used. This must
-  be  a positive non-zero integer. The default is 1000. See :ref:`Pywbemcli and the
-  DMTF pull operations` for more information on pull operations.
-* **--mock-server/-m** - Defines one or more files that define a mock server that
-  can be used to define a mock WBEM server in the pywbemcli process so that
-  pywbemcli commands without access to a real server. When this option is used
-  to define a WBEM server the security options (ex. ``--user``) are irrevalent;
-  they may be included but are not used.
-
-  The following example creates a mock server with two files defining the
-  mock data, shows what parameters are defined for the connection, and then
-  saves that connection named ``mymockserver``:
 
-  .. code-block:: text
+.. _`--user general option`:
+
+--user general option
+^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--user/-u`` is a string that is the user name on the WBEM
+server if a user name is required by the WBEM server to authenticate the client.
+
+.. _`--password general option`:
+
+--password general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--password/-p`` general option is a string that is the
+password for the WBEM server. This option is normally required if the
+:ref:`--user general option` is defined.  If passwords are saved into the
+:term:`connections file` they are not encrypted in the file.
+
+If the WBEM operations performed by any pywbemcli command require a password,
+the password is prompted for if the ``--user general option`` is set (in both
+modes of operation) and the ``--password general option`` is not set.
```

Comment:
Move "general option" out of the backquoted text.

Reply:
``--password`` general option